packages/dds/task-manager/src/taskManager.ts

```diff
@@ -770,7 +770,23 @@ export class TaskManager extends SharedObject<ITaskManagerEvents> implements ITa
 		}
 	}
 
-	public applyStashedOp() {
-		// do nothing...
+	protected applyStashedOp(content: any): void {
+		const taskOp: ITaskManagerOperation = content;
```

Comment:
For TaskManager, we never want to reattempt an action (i.e. never want to reattempt sending ops) after a disconnect/reconnect.  I see your comment in the PR description about "any ops that can be ignored should be ignored" - what's the right way to ignore all the ops?

Reply:
you can only ignore in resubmit. stashed ops my already exist remotely. if they do not exist remotely they will go through resubmit where they can be dropped